data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,58 @@ mission "Arachi Cartoonist 2"
+			`	As he leaves, the curtains open to a set of an Arach street, where a child is purchasing some form of candy. From behind a building in the set, comes an Arach wearing tall, blue and white "socks," as trumpets and tubas start playing an obnoxious track. He runs for the child and knocks the candy to the ground, as he makes some gargling sound. The Arach beside you sighs, saying that "Gibberish, it is. Nonsense, and words best left untranslated." The Arach actor goes on to cling to the walls of the building and grab things from inside the candy shop, throwing them wildly. By now, many well dressed patrons, most likely members of house Bebliss, are getting up and leaving the theater. Meanwhile, a group of Arachi on one of the balconies doesn't even try to muffle their laughter.`
+			`	The scene continues, as a couple comes to the set, walking down the street. As they're about to do what looks like a kiss, the Arach leaps from the building, landing on the bachelor and stealing the kiss, at which point the tubas and trumpets blast as loudly as they can. After a while, the authorities show up, only for the Arach to make a run for it down the street, as the orchestra plays without coordination, and the curtains close. Not many in the audience manifest themselves, most sighing or getting ready to leave, save for the group that was laughing earlier, who are very vocal in announcing their appreciation for the play. "Know of subtetly, house Chamgar does not," the Arach beside you says. "A shame it is, that tainted the soiree with this rivalry, they have."`
+			`	The doors open once again, as new patrons cycle in to watch the third play.`
+			choice
+				`	(Stay for the next play.)`
+				`	(Leave.)`
+					goto leaveearly
+			`	Everyone takes their seats, and the Arach announcer heads to the stage again. "A cathartic tragedy, our third play of this evening is. With you: 'Til Death."`
+			`	This time, all lights inside the theater are turned off, as the curtains open to a single spotlight, on the right side of the stage, highlighting an old Arach inside a small shack, getting dressed. He coughs, and a piano begins to play a melancholic melody. Once he leaves the shack, it's left in the dark, as the spotlight focuses on the middle of the stage, an empty street. Once more, the spotlights shift focus, finally lighting up the left side of the stage, where the Arach starts tending to a garden by the entrance of a manor. The piano stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it's shriveled and dead, plucking it and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`
```

Comment:
Theatre lights are usually turned off, the stage lights would be the relevant ones. I've also swapped it out to an organ approximation. An actual piano, or something too similar to it would be almost, if not impossible to play for an Arach, but a common joke about an organ is that it needs 4 feet to play it.  Plus some general stagecraft alterations and grammar.
```suggestion
			`	The stage lights go dark as the curtains open to a single spotlight. On the right side of the stage it illuminates an old Arach getting dressed inside a small shack. He coughs, and a something akin to an organ begins to play a melancholic melody. Once he leaves the shack it's left in the dark, as the spotlight follows him to the middle of the stage where he makes his way carefully down an empty street. He is allowed to pass beyond the spotlight which goes dark as the perspective shifts and a second spotlight renews the focus, finally lighting up the left side of the stage where the Arach starts tending to a garden by the entrance of a manor. The piano stops its weeping every once in a while, right when the Arach is coughing, only to resume once he's done. Leaving the manor, a richly dressed Arach points to one of the flowers in the garden, showing it to be shriveled and dead, plucking it and throwing it to the ground before leaving. The gardener picks up the flower, sticking it inside his clothes, and continues his work.`
```